### PR TITLE
docs: Eject Hardened JavaScript docs

### PR DESCRIPTION
**Pull request by kriskowal**
The ejected docs will reappear in the Endo repository.

- docs: Eject Hardened JavaScript architecture topics
- fix: Vue config whitespace

### CODE REVIEW

**Comment by kriskowal** [comment]
Pending upon https://github.com/endojs/endo/pull/965 and repointing references.

File: main/guides/js-programming/hardened-js.md [hunk 1]
```diff
@@ -263,7 +263,7 @@ or not:
 
 In compartments used to load Agoric smart contracts,
 `globalThis` is hardened, following OCap discipline.
-These compartments have `console` and `assert` globals from [SES](./ses/).
+These compartments have `console` and `assert` globals from [Hardened JavaScript](https://github.com/endojs/endo/blob/master/packages/ses/README.md).
```
Comment (dckc) [inline main/guides/js-programming/hardened-js.md:266]: We're in the/a hardened js guide already. How about "the `ses` package" for the link text?

**Review by dckc — CHANGES_REQUESTED** [review]
a link to "Hardened JavaScript" when we're already in a Hardened JavaScript guide doesn't seem right.

**Review by dckc — APPROVED** [review]
eject away

**Pull request merged**